Added Function for Saving

msekatchev opened the pull request:
closes #27 
- Created `saveFile` function for overwrite testing, called from `annotate_img` and `annotate_dir`
closes #26 
- Now creates visible image mask in addition to standard `[0,1,2]` mask

msekatchev committed:
Added Function for Saving

closes #27 
- Created `saveFile` function for overwrite testing, called from `annotate_img` and `annotate_dir`
closes #26 
- Now creates visible image mask in addition to standard `[0,1,2]` mask

pdeperio commented on annotate.py:
```diff
@@ -3,15 +3,20 @@
 import numpy as np
 import shutil
 
+#annotate_img("test/B.jpg","MS",10,120)
```
Why did this come back?

msekatchev replied:
Sorry about that, I put that there for easy access and forgot to remove it

pdeperio replied:
is it not stored in your command history (when you press ↑ in terminal)?

msekatchev replied:
True, the way I have it set up with Sublime Text a new terminal runs every time so the commands are refreshed...
Not the most efficient, but on the other hand I can quickly run the code with a shortcut.

pdeperio commented on annotate.py:
```diff
@@ -25,54 +30,47 @@ def draw_circle(event,x,y,flags,param):
-# s to close image and exit program
-# r to write coordinates (you will be prompted for the PMT feature ID if a PMT has not been selected yet)
-# f to select new PMT
-# n to increment the featureID number
```
Lost documentation of incrementing?

pdeperio commented on annotate.py:
```diff
@@ -25,54 +30,47 @@ def draw_circle(event,x,y,flags,param):
+        ##size2 - size of the 2nd brush circle, in px
+        ##initials - First name and last name initials of the person doing the labelling. Recorded next to every PMT feature ID line in the text file.
+    ##outputs
+        ##label.jpg - A map of all the features in binary (white on black)
```
Seems like older version of documentation got injected here?

msekatchev replied:
Hmm yeah can't remember how that happened lol
Sorry about that I'll make a new commit

pdeperio reviewed (changes requested):
Sorry for delay, slipped my mind, thanks for reminder. 

Just some small comments on documentation.

msekatchev committed:
Made changes to comments

Removed unneeded comments, restored newer comments that were lost during code update.
